marble/config/config.go

```diff
@@ -7,14 +7,29 @@
 // Package config defines the environment variables expected by the Marble for configuration settings.
 package config
 
+import (
+	"path/filepath"
+
+	"github.com/edgelesssys/marblerun/util"
+)
+
 // CoordinatorAddr is the marble's addr to connect to the coordinator via gRPC
 const CoordinatorAddr = "EDG_MARBLE_COORDINATOR_ADDR"
 
+// CoordinatorAddrDefault is the marble's default addr to connect to the coordinator via gRPC
+const CoordinatorAddrDefault = "localhost:2001"
+
 // Type is the marble's type used for attestation with the coordinator
 const Type = "EDG_MARBLE_TYPE"
 
 // DNSNames are the alternative dns names for the marble's certificate
 const DNSNames = "EDG_MARBLE_DNS_NAMES"
 
+// DNSNamesDefault returns the default alternative dns names for the marble's certificate
+func DNSNamesDefault() string { return util.MustGetenv("EDG_MARBLE_TYPE") }
```

Comment:
Should we also include localhost here? E.g., to simplify the samples' Readmes. (also @m1ghtym0)

Reply:
Sounds good to me

Reply:
I do not see how this would simplify the samples, since the EDG_MARBLE_TYPE does not have a default value and needs to be set anyway, there should not be a difference for the sample between using this type or "localhost". Or am I missing something?

Reply:
I think we need  to have localhost as one of the DNS names for the samples to work outside k8s, so the user would need to set both MARBLE_TYPE and DNS_NAMES.